frontend/components/text-classifier/results/RecordTextClassification.vue

```diff
@@ -90,7 +90,9 @@ export default {
         this.dataset.labels.length === 2 && !this.dataset.isMultiLabel;
       return (
         this.record.status !== "Validated" &&
-        !isBinary &&
+        (!isBinary ||
+          (isBinary && this.record.annotation) ||
+          this.record.prediction) &&
```

Comment:
Could we just remove these conditions? The cases where you can validate records using the `Validate` button should be:

1. has annotation  OR
2. has prediction OR
3. is a multi_label record

Those cases are covered by lines 96, 97 and 98

Reply:
absolutely right